File: C++/First Steps in Coding - Basics of C++/First steps/Projects Creation.cpp
#include <iostream>
#include <cstdlib>
using namespace std;
int main() {
	string architect_name; int projects, required_time;
 	cin>>architect_name; cin>>projects;
    required_time=projects*3;
  	cout<<"The architect "<<architect_name<<" will need "<<required_time<<" hours to complete "<<projects<<" project/s.";
  return 0;
}

File: Go/First Steps Basics/Projects Creation.go
package main

import "fmt"

func main() {
	var archName string
	var projects, totalTime, timePerProject int
	timePerProject = 3
	fmt.Scanln(&archName)
	fmt.Scanln(&projects)
	totalTime = timePerProject * projects
	output := fmt.Sprintf("The architect %v will need %v hours to complete %v project/s.", archName, totalTime, projects)
	fmt.Println(output)
}
